drivers/gpio/gpio_stellaris.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Do we need this file, can we just merge it with gpio_stelaris.c?

Reply:
fixed